azurerm/resource_arm_container_registry_migrate_test.go

```diff
@@ -93,11 +95,11 @@ func TestAccAzureRMContainerRegistryMigrateState(t *testing.T) {
 	}
 }
 
-func createResourceGroup(client *ArmClient, resourceGroupName string, location string) error {
+func createResourceGroup(client *ArmClient, ctx context.Context, resourceGroupName string, location string) error {
```

Comment:
can you make `ctx` the first param here?

Reply:
fixed